nengo_gui/monkey.py

```diff
@@ -67,12 +71,20 @@ def determine_line_number(filename='<string>'):
         return line
     return None
 
+stdout = StringIO()
 
 @contextlib.contextmanager
 def patch():
+    global stdout
```

Comment:
Instead of using `global`, perhaps consider implementing this as a context manager class? `contextlib.contextmanager` is cool, but needing to store something is probably the right time to switch over just implementing `__enter__` and `__exit__` methods.